static/css/_attractions-theme.scss

```diff
@@ -1,62 +1,17 @@
-$font: 'IBM Plex Sans', sans-serif;
-$header-font: 'Montserrat', sans-serif;
-
+@use "sass:color";
 $main: #007a00;
```

Comment:
Why is this one in particular left outside?

Reply:
It is used in the file itself later on, for the glowing shadow